compareECLFiles.cmake

```diff
@@ -12,6 +22,8 @@ set(BASE_RESULT_PATH ${PROJECT_BINARY_DIR}/tests/results)
 # Input:
 #   - casename: basename (no extension)
 #
+# Details:
+#   - This test class compares output from a simulation to reference files.
 macro (add_test_compareECLFiles casename filename simulator)
```

Comment:
I would either:
1. Split the `add_test_compareECLFiles( )` into `add_test_compareSummaryFiles( )`, `add_test_compareRestartFiles( )` and `add_test_compareInitFiles( )`
2. Or alternatively just include the INIT file testing in the `add_test_compareECLFiles( )`

The current split of INIT files as a separate test and then joining (the much larger context) of summary restart files in one test does not make sense.